music/src/core/data_test.ts

```diff
@@ -15,7 +15,10 @@
  * limitations under the License.
  */
 
-import * as tf from '@tensorflow/tfjs-core';
+import * as tf from '@tensorflow/tfjs';
+
+// To remove the warning that tf.get() is deprecated.
```

Comment:
why not just stop using tf.get?

Reply:
That's what I was trying to explain in the PR description. I can only find 2 instances of `.get()`, in this file. Even if I remove those tests, I still get all the errors (read: hundreds), and I can't find where the other instances are. I searched in files for any `.get(`, and we only use it to get from a `Map`....

Reply:
Ok so I tried to replace the two `.get()` calls in this function but `arraySync` seems to be busted for scalars, so I filed https://github.com/tensorflow/tfjs/issues/1271.

Reply:
Oh apparently 0.15.3 fixes this so uhhh hold.

Reply:
fixed!